### DISCUSSION TITLE
Assistance in running nexpy via Python

### DISCUSSION THREAD

**Discussion by apostolos-vagias** [nexpy/nexpy#315] [Q&A]
Dear colleagues,

I need the help of your expertise. I have installed nexpy and all associated libraries indicated as prerequisites:
https://nexpy.github.io/nexpy/includeme.html

 My current Python version is 3.8.3. I try to run nexpy either via Anaconda or via cmd directly in Python. What I get is the following error, which I would appreciate if someone has an idea how to make GUI appear properly.  

![Nexpy_error_message_4thFeb2021](https://user-images.githubusercontent.com/78554531/106926592-88134000-6711-11eb-816e-fe70efaf0ef9.png)
![Nexpy_error_message_4thFeb2021](https://user-images.githubusercontent.com/78554531/106926670-9eb99700-6711-11eb-9bea-d74e3d3621ec.png)

I would need nexpy in order to load some "nxs" files and further process them. Any ideas how to troubleshoot this would be great!

Many thanx in advance,
Apostolos

**Comment by rayosborn** [comment]
Is this on Windows? There is a similar report here - https://github.com/jupyter/qtconsole/issues/400. If that looks like the same issue, I will look into how they fixed it.

**Reply by rayosborn** [reply]
If this is indeed the issue, it seems as if it is associated with the tornado module (https://github.com/jupyter/notebook/pull/5933). A recent suggestion is to upgrade tornado to v6.1. Unfortunately, I am unable to test things on Windows.

**Reply by apostolos-vagias** [reply]
Dear Ray: Thank you very much for your prompt response. Yes, the issue is on Windows 10 (64-bit). I will have a look and post again, if the issue persists. All the best!

**Reply by apostolos-vagias** [reply]
Ray, many thanx! Your suggestion has been crucial for this troubleshooting. Indeed, tornado v6.1 installation (not from anaconda, but from here: https://pypi.org/project/tornado/) solved this problem for my Windows 10 (64-bit) machine.

**Reply by rayosborn** [reply]
I'm pleased it has worked out. I will look into adding a note in the online documentation. I work with someone who also runs Windows, but I don't myself so it's useful to get this kind of feedback.
